challenge_2/python/tavery321/src/c2.py

```diff
@@ -0,0 +1,8 @@
+arr = ['a', 'a', 'b','b', 1, 2, 2, 3, 3, 'z', 'z']
+answer = ''
+for i in range(len(arr)):
+	if(arr.count(arr[i]) == 1):
```

Comment:
Your solution should run in O(n) time; this runs in O(n^2). Reconsider your usage of `arr.count(arr[i])` python has to look through your entire array to do this. Is there any way you can count all the elements with a single iterate of `arr`?